Calculator/index.js

```diff
@@ -1,8 +1,105 @@
 function addEventListenerToButton(element){
     element.addEventListener('click', function(){
-        console.log(element.getAttribute("data-button-value"))
+        let value=element.getAttribute("data-button-value")
+        let type=element.getAttribute("data-button-type")
+
+        switch(type){
+            case "clear": 
+
+                clear();
+                document.getElementById("old-inputs").innerHTML = '';
+                document.getElementById("inputs").innerHTML = '';
+            break;
+
+            case "delete":
+                del();
+            break;
```

Comment:
I think there's not a clear separation of concerns here, and also, you're using different criteria for different pieces of code that look fairly similar. For example, in the case for `"clear"`: you have a `clear()` function that you're executing and then you're changing the view, right? You have a similar structure in the `del()` function, but you put the UI management _inside_ the function, also. I think I'd rather the second approach if I had to choose between those options, but still. 
I would also rather having a function called something like `updateUI()` or updateUI({...options})` to update the UI
I think that modification can wait until a React refactor for this code, but I didn't want to miss the chance to let you know that interacting each time differently between the UI and the data model can lead to some nasty bugs